src/toast/toast.tsx

```diff
@@ -19,11 +19,14 @@ import {
 } from "./toast.styles";
 import { ToastProps } from "./types";
 
+const defaultAutoDismissTime = 4000;
```

Comment:
bit nitpicky but can declare this const in uppercase
```suggestion
const DEFAULT_AUTO_DISMISS_TIME = 4000;
```